README.md

```diff
@@ -122,6 +122,10 @@ models = [
   "Dlib", 
   "SFace",
   "GhostFaceNet",
+  "yolov8",
```

Comment:
yolo is only for face detection now, do not mention it in facial recognition models